server/conn_stmt.go

```diff
@@ -310,14 +313,18 @@ func parseStmtFetchCmd(data []byte) (uint32, uint32, error) {
 	return stmtID, fetchSize, nil
 }
 
-func parseExecArgs(sc *stmtctx.StatementContext, args []types.Datum, boundParams [][]byte, nullBitmap, paramTypes, paramValues []byte) (err error) {
+func parseExecArgs(sc *stmtctx.StatementContext, args []types.Datum, boundParams [][]byte,
+	nullBitmap, paramTypes, paramValues []byte, enc *inputDecoder) (err error) {
 	pos := 0
 	var (
 		tmp    interface{}
 		v      []byte
 		n      int
 		isNull bool
 	)
+	if enc == nil {
```

Comment:
When we will go this logic？It seems that if `chs` is "", enc is `UTF8Encoding`.

Reply:
This is used to prevent the misuse of this function.